rules/precompiled_apple_resource_bundle.bzl

```diff
@@ -90,6 +94,7 @@ def _precompiled_apple_resource_bundle_impl(ctx):
         ),
         rule_label = fake_rule_label,
         version = None,
+        **partials_args
```

Comment:
Can you name this something slightly different than the dict var that you're creating for ease of understanding?

Reply:
I like that!

Reply:
Updated, and added comment for why this needs to be `False`